ansible_collections/arista/avd/examples/l2ls-fabric/group_vars/DC1/dc1.yml

```diff
@@ -0,0 +1,12 @@
+---
+### group_vars/DC1/dc1.yml
+
+# Set the Fabric Name - must match an Ansible Inventory Group
+fabric_name: DC1
+
+#### Override for vEOS/cEOS Lab Caveats ####
+p2p_uplinks_mtu: 1500
```

Comment:
MTU 1500 is wrong! Please remove `#### Override for vEOS/cEOS Lab Caveats ####`. There are no caveats. Let's switch all examples to `9214`.

Reply:
Agree. Minor PR is the best way as we have this in multiple places. #5706